static/templates/pm_list_item.hbs

```diff
@@ -13,7 +13,7 @@
             {{recipients}}
             {{> status_emoji status_emoji_info}}
         </a>
-        <span class="unread_count {{#if is_zero}}zero_count{{/if}}">
+        <span id="private_message_unread_count" class="unread_count {{#if is_zero}}zero_count{{/if}}">
```

Comment:
This needs to be a CSS class, not an ID -- HTML spec requires that IDs be unique in a document, and this is a thing one has more than one of.

Reply:
yes, i agree, Updated the PR.
Thanks :)